apps/api-server/src/db.js

```diff
@@ -26,17 +26,33 @@ if (dbConfig.mysqlSTGeoMode == 'on') {
 const dialectOptions = {
 	charset            : 'utf8',
 	multipleStatements : dbConfig.multipleStatements,
-	socketPath         : dbConfig.socketPath
+	socketPath         : dbConfig.socketPath,
+	ssl: {
+		require: dbConfig.requireSsl
+	}
 }
 
 if (dbConfig.MYSQL_CA_CERT && dbConfig.MYSQL_CA_CERT.trim && dbConfig.MYSQL_CA_CERT.trim()) {
```

Comment:
I would expect the ssl object to be set before the dialectOptions and then incorporated into the dialectOptions immediately instead of afterwards.

```

const ssl = { 
  require: dbConfig.requireSsl
};
if (dbConfig.MYSQL_CA_CERT?.trim?.()) {
  ssl.rejectUnauthorized = true;
  ssl.ca = [ dbConfig.MYSQL_CA_CERT ];
}
const dialectOptions = {
  charset            : 'utf8',
  multipleStatements : dbConfig.multipleStatements,
  socketPath         : dbConfig.socketPath,
  ssl,
}
```

Or if you want to be fancy you can combine the ssl instantiation but you have to ask yourself if this is more readable
```
const ssl = { 
    require: dbConfig.requireSsl,
    ...(dbConfig.MYSQL_CA_CERT?.trim?.() ? {
        rejectUnauthorized: true,
        ca: [ dbConfig.MYSQL_CA_CERT ],
    } : {})
}
```

Reply:
I implemented the first one, I think the latter is a bit too fancy and indeed not very readable